core/domain/stats_services.py

```diff
@@ -571,6 +571,11 @@ def get_updated_exp_issues_models_for_new_exp_version(
         if 'state_names' in playthrough.issue_customization_args:
             state_names = (
                 playthrough.issue_customization_args['state_names']['value'])
+            # Here we used assert to narrow down the type from various allowed
```

Comment:
But you're just narrowing it down to list, not List[str]? Ditto elsewhere.

Should it be more explicit?

Reply:
Done, instead of assertions now cast is used.